source/documentation/deploying_services/index.md

```diff
@@ -273,10 +275,21 @@ where `SERVICE_NAME` is a unique descriptive name for this service instance, and
 cf update-service my-pg-service -p small-ha-9.5
 ```
 
-The plan upgrade will begin immediately and will usually be completed within about an hour. You can check the status of the change by running the `cf services` command.
+The plan upgrade will begin immediately and will complete within about an hour. You can check the status of the change by running the `cf services` command.
 
 You can also [queue a plan upgrade](/deploying_services.html#queue-a-plan-migration-postgresql) to happen during a maintenance window to minimise service interruption.
 
+#### Different encryption type
+
+You cannot upgrade your service plan using the `cf update-service` command if the new plan has a different encryption type to the current plan, for example from free unencrypted to paid high availability encrypted.
+
+To upgrade, you must set up a new service and move your app data over.
+
+1. [Set up a new PostgreSQL service](/deploying_services.html#set-up-a-postgresql-service) with a new plan that is a encryption type to your current plan.
```

Comment:
> with a new plan that is a encryption type to your current plan.

I think maybe you mean *that has a different encryption type to your current plan* ?

Reply:
I am seriously losing my mind. Apologies and thanks!